src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -312,6 +319,39 @@ export class BuilderNavbarComponent implements OnDestroy {
     this.store.cancelSubmission();
   }
 
+  /**
+   * Create a new changelog for the active learning object
+   */
+  updateChangelog() {
+    if (this.changelog) {
+      this.changelogService.createChangelog(this.store.learningObjectEvent.getValue().author.id,
+        this.store.learningObjectEvent.getValue().id,
+        this.changelog)
+          .then(() => {
+          this.showSubmission = false;
+          this.page = 1;
+        }).catch(e => {
+          console.error(e);
+          this.showSubmission = false;
+          this.page = 1;
+          this.toasterService.notify('Error!', e, 'bad', 'far fa-times');
+        });
+    } else {
+      this.showSubmission = false;
+      this.page = 1;
+    }
+    this.submitForReview(this.submitCollection);
+  }
+
+  /**
+   * 
```

Comment:
Why is there no comment here?

![image](https://user-images.githubusercontent.com/7431390/55911214-24b83580-5bae-11e9-848b-b8e729875626.png)

Reply:
Fixed.